doc/Examples/openroad-sky130.md

```diff
@@ -0,0 +1,94 @@
+# Introduction with Sky130 and OpenROAD
+
+The following directions will get a simple ``pass`` design from RTL to GDS using the [OpenROAD tools](https://theopenroadproject.org) and the [Skywater 130nm PDK](https://github.com/google/skywater-pdk). These directions are meant to provide the minimal set of steps to do so, please reference the next section, [Hammer End-to-End Integration Tests](https://hammer-vlsi.readthedocs.io/en/stable/Examples/e2e.html), for more detailed descriptions of all files and commands.
+
+## Instructions
+
+First, follow the [Hammer Developer Setup](https://hammer-vlsi.readthedocs.io/en/stable/Hammer-Basics/Hammer-Setup.html#developer-setup) to clone Hammer and install/activate the poetry virtual environment.
+
+Next, run the setup script to install the OpenROAD tools using Conda, and Skywater 130 nm PDK using the [Open-PDKs tool](https://github.com/RTimothyEdwards/open_pdks).
+This step will take a long time due to the amount and size of the required installs.
+
+```shell
+cd hammer/e2e
+./scripts/setup-sky130-openroad.sh [INSTALL_PREFIX]  # default = ~/
```

Comment:
`INSTALL_PREFIX` needs explanation. `PREFIX` is probably better (like compiling a program from source).

Reply:
You're right, just added!